### PR TITLE
Allow wcommon 2

### PR DESCRIPTION
**Pull request by robbecker-wf**
## Motivation
We'd like to be able to upgrade to w_common v2

## Changes
Replace deprecated w_common calls with their equivalent v2 calls. These calls also exist in v1 so we can go ahead and leave these in for a patch release and raise the w_common max to allow w_common v2.

## Review
_[See CONTRIBUTING.md][contributing-review-types] for more details on review types (+1 / QA +1 / +10) and code review process._

  

Please review: 

### QA Checklist
- [ ] Tests were updated and provide good coverage of the changeset and other affected code
- [ ] Manual testing was performed if needed
    - [ ] Steps from PR author: 
        
    - [ ] Anything falling under manual testing criteria [outlined in CONTRIBUTING.md][contributing-manual-testing]

## Merge Checklist
While we perform many automated checks before auto-merging, some manual checks are needed:
- [ ] A Frontend Frameworks Design member has reviewed these changes
- [ ] There are no unaddressed comments _- this check can be automated if reviewers use the "Request Changes" feature_
- [ ] _For release PRs -_ Version metadata in Rosie comment is correct

[contributing-review-types]: https://github.com/Workiva/over_react/blob/master/CONTRIBUTING.md#review-types
[contributing-manual-testing]: https://github.com/Workiva/over_react/blob/master/CONTRIBUTING.md#manual-testing-criteria

### CODE REVIEW

**Comment by aviary-wf** [comment]
## Security Insights
No security relevant content was detected by automated scans.

## Action Items
* Review PR for [security impact](https://wiki.atl.workiva.net/display/SECURITY/Development+Security+Review+Guidelines); comment "security review required" if needed or unsure
* Verify [`aviary.yaml`](https://wiki.atl.workiva.net/pages/viewpage.action?pageId=42762752#SecurityInsightFramework&Tooling-ConfiguringAviary) coverage of security relevant code
---
_Questions or Comments? Reach out on Slack: #support-infosec._

**Comment by robbecker-wf** [comment]
Ran a separate branch to run CI using w_common v2 https://github.com/Workiva/over_react/pull/777

File: app/over_react_redux/todo_client/pubspec.yaml [hunk 1]
```diff
@@ -32,3 +32,4 @@ dev_dependencies:
 dependency_overrides:
   over_react:
     path: '../../../'
+  w_common: ^2.0.0
```
Comment (robbecker-wf) [inline app/over_react_redux/todo_client/pubspec.yaml:35]: hmmm not sure this should be there

Comment (greglittlefield-wf) [inline app/over_react_redux/todo_client/pubspec.yaml:35]: Was this meant to be committed?

**Review by greglittlefield-wf — APPROVED** [review]
Just one question, otherwise LGTM.

- Changes look good
- CI passes in test branch

+10

File: app/over_react_redux/todo_client/pubspec.yaml (hunk 1 above)
Comment (robbecker-wf) [reply app/over_react_redux/todo_client/pubspec.yaml]: nope! removed.

**Review by greglittlefield-wf — APPROVED** [review]
+10

**Comment by greglittlefield-wf** [comment]
Closing and reopening to re-trigger Rosie

**Comment by greglittlefield-wf** [comment]
@rmconsole-wf

**Review by greglittlefield-wf — APPROVED** [review]
+10

**Comment by robbecker-wf** [comment]
@Workiva/release-management-p

**Review by rmconsole-wf — APPROVED** [review]
+1 from RM

**Pull request merged**